src/ui/components/DropdownMenu.tsx

```diff
@@ -1,20 +1,33 @@
 import { ChangeEvent, FC, useEffect, useState, useCallback } from "react";
-import { Form } from "react-bootstrap";
+import { Form, Toast, ToastContainer } from "react-bootstrap";
 
 type DropdownMenuProps = {
   onSelectFeatureConfigId: (featureId: string) => void;
 };
 
 const DropdownMenu: FC<DropdownMenuProps> = ({ onSelectFeatureConfigId }) => {
   const [featureConfigs, setFeatureConfigs] = useState([]);
+  const [showToast, setShowToast] = useState(false);
+  const [toastMessage, setToastMessage] = useState("");
+  const [toastVariant, setToastVariant] = useState<"success" | "danger">(
+    "danger",
```

Comment:
Instead of defining this all every time we need to handle errors, we should have a re-usable component for this.

Take a look at this implementation I added to another react webapp:

https://github.com/mozilla/limelight/blob/main/src/hooks/useToasts.ts
https://github.com/mozilla/limelight/blob/main/src/components/Toasts.tsx

This pattern allows you to define a "custom context" and "context provider". The context object holds everything to manage toasts and the context provdider makes the context available to all child components.

See https://github.com/mozilla/limelight/blob/main/src/components/App/index.tsx#L50 for usage of Toast.Provider

See https://github.com/mozilla/limelight/blob/5015f0be94be6e927747c40665fcdba59a79d8bb/src/components/NewEditImportPane/ExperimentImportMessageForm.tsx#L238 for an example of adding toasts via the context object.